src/textual/widget.py

```diff
@@ -1580,3 +1577,22 @@ def key_pageup(self) -> bool:
             self.scroll_page_up()
             return True
         return False
+
+
+def _get_rich_justify(css_align: str) -> JustifyMethod:
+    """Given the value for CSS text-align, return the analogous argument
+    for the Rich text `justify` parameter.
+
+    Args:
+        css_align: The value of text-align CSS property.
+
+    Returns:
+        JustifyMethod: The Rich JustifyMethod that corresponds to the text-align
+            value
+    """
+    assert css_align in VALID_TEXT_ALIGN
+    return {
+        "start": "left",
+        "end": "right",
+        "justify": "full",
```

Comment:
Since Textual doesn't support RTL, we can just directly map start -> left and so on.